scipy/signal/bsplines.py

```diff
@@ -133,6 +133,8 @@ def gauss_spline(x, n):
     """Gaussian approximation to B-spline basis function of order n.
     """
     signsq = (n + 1) / 12.0
+    if signsq == 0:
```

Comment:
This can only happen when `n = -1`  and doesn't make sense anyways.  And the error to raise is wrong input argument not zero by division anyways.

Reply:
Fair point, I wrote this to conform to the tests, but you are correct that that makes more sense. I'll fix this.

Reply:
This is still not quite right. Rather than doing input validation for `n`, I'd just remove these two lines again and instead just document that `n` needs to be an integer >= 0.

Reply:
You could also put in a reference to https://pdfs.semanticscholar.org/360c/77005e0ac26fa0aecbea214f37609bb52414.pdf, that contains the math. It has `sqrt(n/12)` rather than `sqrt((n+1)/12))`, but I don't want to read the paper in enough detail to sort that out. Just throwing the link into the docstring under `References` would be helpful though.